lifecycle/lifecycle-viewmodel-savedstate/build.gradle

```diff
@@ -22,61 +22,55 @@
  * modifying its settings.
  */
 
-import androidx.build.JetbrainsAndroidXPlugin
+import androidx.build.LibraryType
 import androidx.build.PlatformIdentifier
-import androidx.build.Publish
-import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType
 
 plugins {
     id("AndroidXPlugin")
     id("com.android.library")
+    alias(libs.plugins.kotlinSerialization)
     id("JetbrainsAndroidXPlugin")
 }
 
-JetbrainsAndroidXPlugin.applyAndConfigure(project)
-
 androidXMultiplatform {
     android()
     desktop()
-    mac()
-    linux()
     ios()
+    linux()
+    mac()
     js()
     wasm()
 
     defaultPlatform(PlatformIdentifier.ANDROID)
-}
-
-kotlin {
-    js {
-        browser()
-    }
-    wasmJs()
-
-    watchosArm64()
-    watchosArm32()
-    watchosX64()
-    watchosSimulatorArm64()
-    tvosArm64()
-    tvosX64()
-    tvosSimulatorArm64()
 
     sourceSets {
         commonMain {
             dependencies {
                 api("org.jetbrains.compose.annotation-internal:annotation:1.7.1")
-                api(project(":core:core-bundle"))
-                api(project(":lifecycle:lifecycle-common"))
+                api(project(":savedstate:savedstate"))
                 api(project(":lifecycle:lifecycle-viewmodel"))
-                api("org.jetbrains.androidx.savedstate:savedstate:1.2.2")
+                implementation("org.jetbrains.androidx.core:core-bundle:1.1.0-alpha01")
```

Comment:
Why do we still need this dependency here?

Reply:
because this is a lifecycle module and there are some deprecated but public APIs with core-bundle.
I will clean it after the current PR